typings/codePush.d.ts

```diff
@@ -299,14 +299,19 @@ declare enum InstallMode {
  */
 interface InstallOptions {
     /**
-     * Used to specity the InstallMode used for the install operation. This is optional and defaults to InstallMode.ON_NEXT_RESTART.
+     * Used to specify the InstallMode used for the install operation. This is optional and defaults to InstallMode.ON_NEXT_RESTART.
      */
     installMode?: InstallMode;
     
     /**
      * If installMode === ON_NEXT_RESUME, the minimum amount of time (in seconds) which needs to pass with the app in the background before an update install occurs when the app is resumed.
      */
     minimumBackgroundDuration?: number;
+    
+    /**
+     * Used to specify the InstallMode used for the install operation if the update is mandatory. This is optional and defaults to InstallMode.IMMEDIATE.
+     */
+    mandatoryInstallMode?: InstallMode;
```

Comment:
Just spoke to @lostintangent, this is what he wants. He was under the impression that mandatory updates were already immediate, but that something must have changed to remove that bug/feature lol